docs/sphinx/user_guide/feature/reduction.rst

```diff
@@ -291,22 +291,58 @@ any number of ``RAJA::expt::Reduce`` objects to the ``RAJA::forall`` method::
   std::cout << rm_loc.getLoc() ...
 
 Again, the lambda expression parameters are in the same order as
-the ``RAJA::expt::Reduce`` arguments to ``RAJA::forall``. Both the types and 
-order of the parameters must match to get correct results and to compile 
+the ``RAJA::expt::Reduce`` arguments to ``RAJA::forall``. Both the types and
+order of the parameters must match to get correct results and to compile
 successfully. Otherwise, a static assertion will be triggered::
 
   LAMBDA Not invocable w/ EXPECTED_ARGS.
 
 .. note:: This static assert is only enabled when passing an undecorated C++
-          lambda. Meaning, this check will not happen when passing 
+          lambda. Meaning, this check will not happen when passing
           extended-lambdas (i.e. DEVICE tagged lambdas) or other functor like
           objects.
 
 .. note:: The experimental ``RAJA::forall`` interface is more flexible than the
           current implementation, other optional arguments besides
           ``RAJA::expt::Reduce`` can be passed to a ``RAJA::forall`` to extend
-          its behavior. In the above example we demonstrate using 
-          ``RAJA::expt::KernelName``, which wraps a ``RAJA::forall`` executing 
-          under a ``HIP`` or ``CUDA`` policy in a named region. Use of 
+          its behavior. In the above example we demonstrate using
+          ``RAJA::expt::KernelName``, which wraps a ``RAJA::forall`` executing
+          under a ``HIP`` or ``CUDA`` policy in a named region. Use of
           ``RAJA::expt::KernelName`` does not require an additional
           parameter in the lambda expression.
+
+
+Experimental reduction support in Launch
+........................................
+
+The experimental reduction interface is also supported with the ``RAJA::launch`` API.
+The usage of the experiemental reductions is similar to the forall example as illustrated below::
+
+  double* a = ...;
+
+  double rs = 0.0;
+  double rm = 1e100;
+
+  RAJA::launch<EXEC_POL> ( Res,
+  RAJA::expt::Reduce<RAJA::operators::plus>(&rs),
```

Comment:
Indentation and vertical alignment of curly braces and parens would help readability.